projects/1.18/assets/mimi-mod/mimi/patchouli_books/guide_book/zh_cn/categories/9commands.json

```
@@ -0,0 +1,6 @@
+{
+	"name": "指令",

```

```suggestion
	"name": "命令",
```
见MinecraftWiki，全文件通查